python/ray/tune/tune.py

```diff
@@ -59,7 +59,7 @@ def run(run_or_experiment,
         verbose=2,
         resume=False,
         queue_trials=False,
-        reuse_actors=True,
+        reuse_actors=False,
```

Comment:
Turning this off because this only affects HyperBand and PBT and breaks things if `reset_config` is not set.